lib/device.js

```diff
@@ -251,6 +256,174 @@ const util = require('util'),
+                        + __pad(curDate.getSeconds(), 2);
+
+                    return dateFormat;
+                }
+
+                function __pad(number, length) {
+                    let str = '' + number;
+                    while (str.length < length) {
+                        str = '0' + str;
+                    }
+                    return str;
+                }
+
+                const fileName = options.session.target.name + "_" +"display" + options.display+ "_" + __getDate() +".png";
+                return fileName;
+            }
+
+            function _captureFile(next) {
+                log.info("device#capture#_captureFile()");
+                const cmd = "/bin/mkdir -p " + options.capDirPath;
+                options.session.run(cmd, null, null, null, function(err) {
+                    if (err) {
+                        return setImmediate(next, err);
+                    } else {
+                        options.targetCapDir = true;
+                        const target = options.session.getDevice(),
+                            addr = target.lunaAddr.captureCompositorOutput,
+                            param = {
+                                // luna param
+                                subscribe: false,
+                                output : options.sourcePath,
+                                format: options.capFormat,
+                                displayId : options.display
+                            };
+
+                        luna.sendWithoutErrorHandle(options, addr, param, function(lineObj, next) {
+                            log.silly("device#capture#_captureFile():", "lineObj:", lineObj);
+
+                            if (lineObj.returnValue) {
+                                log.info("device#capture#_captureFile():","Capture file in target : " + lineObj.output);
+                                next(null, {});
+                            } else {
+                                return setImmediate(next, errHndl.changeErrMsg("FAILED_CALL_LUNA", lineObj.errorText), {});
+                            }
+                        }, next);
+                    }
+                });
+            }
+
+            function _copyToLocal(next) {
+                log.info("device#capture#_copyToLocal()");
+                if (!fs.existsSync(options.destinationPath)) {
+                    mkdirp(options.destinationPath, function(err) {
+                        if (err) {
+                            return setImmediate(next, err);
+                        } else {
+                            pullLib.pull(options, next);
+                        }
+                    });
+                } else {
+                    pullLib.pull(options, next);
+                }
+            }
+
+            function _removeCaptureDir(next) {
+                log.info("device#capture#_removeCaptureDir()");
+                const cmd = '/bin/rm -rf ' + options.capDirPath;
```

Comment:
is /bin/ needed?

Reply:
Yes. usually in CLI, make command, which is use for target device, add /bin/. 
{code}
options.session.run('/bin/rm -f "' + devicePkgPath + '"', null, null, null, next); 
{code}